app/index.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express'
+import bodyParser from 'body-parser'
+import morgan from 'morgan'
+import routes from './routes'
+
+const app = express()
+
+app.use(bodyParser.json())
```

Comment:
I can see we are using express 4.17 hence body parser has been re-added under the methods `express.json()` and `express.urlencoded()`